ras/ras_ppcdiag.py

```diff
@@ -79,8 +79,8 @@ def test_build_upstream(self):
                     self.cancel("Fail to install %s required for this test." %
                                 package)
             url = self.params.get(
-                    'ppcdiag_url', default='https://github.com/power-ras/'
-                    'ppc64-diag/archive/refs/heads/master.zip')
+                'ppcdiag_url', default='https://github.com/power-ras/'
```

Comment:
why is this change require? is this style fixing , if yes i say we drop this from commit as old code looks good

Reply:
yes Praveen when I ran pycodestyle .I think it got fixed automatically. How to drop it from the commit?